pyproject.toml

```diff
@@ -23,7 +23,7 @@ pytz = "^2020.1"
 magma-suite = "1.0.0"
 tibanna-ff = "^1.0.0"
 MarkupSafe = "1.1.1"
-foursight-core = "^1.0.0"
+foursight-core = "^1.0.1.1b44"
```

Comment:
Did we not finally merge th is so we should be using a better version here?

Reply:
Sorry not sure what your exact question is? Using beta of foursight-core because not yet merged.